.vscode/launch.json

```diff
@@ -6,7 +6,7 @@
       "type": "python",
       "request": "launch",
       "program": "library_test.py",
-      "args": ["-p", "<your password>"],
+      "args": ["-f", "library_test.json"],
```

Comment:
```suggestion
      "args": ["-cf", "library_test.json"],
```

Reply:
see below, I would prefer `-c` then (to keep short versions of params at a single letter)